Building krnelci/debos image fails

Running 
```
./kci_docker build --verbose debos --fragment=kernelci
```
fails with the following message

```
Building kernelci/debos:kernelci
Traceback (most recent call last):
  File "./kci_docker", line 136, in <module>
    main(parser.parse_args())
  File "./kci_docker", line 103, in main
    _, build_log = _build_image(dockerfile, name, buildargs)
  File "./kci_docker", line 39, in _build_image
    fileobj=dockerfile_obj, tag=tag_name, buildargs=buildargs
  File "/home/mgalka/projects/kernelci/kernelci-core/venv3.6/lib/python3.6/site-packages/docker/models/images.py", line 285, in build
    raise BuildError(chunk['error'], result_stream)
docker.errors.BuildError: The command '/bin/sh -c go get github.com/go-debos/debos/cmd/debos &&     go install github.com/go-debos/debos/cmd/debos' returned a non-zero code: 2

```

After a bit more investigation the following error messages can be found:

```
Step 10/32 : RUN go get github.com/go-debos/debos/cmd/debos &&     go install github.com/go-debos/debos/cmd/debos
 ---> Running in c1f31bae7899
# golang.org/x/sys/unix
usr/local/go/src/golang.org/x/sys/unix/syscall.go:83:16: undefined: unsafe.Slice
usr/local/go/src/golang.org/x/sys/unix/syscall_unix.go:118:7: undefined: unsafe.Slice
usr/local/go/src/golang.org/x/sys/unix/sysvshm_unix.go:33:7: undefined: unsafe.Slice
# github.com/klauspost/compress/huff0
usr/local/go/src/github.com/klauspost/compress/huff0/decompress.go:774:20: cannot convert out (type []byte) to type *[256]byte
usr/local/go/src/github.com/klauspost/compress/huff0/decompress.go:775:20: cannot convert out[dstEvery:] (type []byte) to type *[256]byte
usr/local/go/src/github.com/klauspost/compress/huff0/decompress.go:776:20: cannot convert out[dstEvery * 2:] (type []byte) to type *[256]byte
usr/local/go/src/github.com/klauspost/compress/huff0/decompress.go:777:20: cannot convert out[dstEvery * 3:] (type []byte) to type *[256]byte
usr/local/go/src/github.com/klauspost/compress/huff0/decompress.go:1013:20: cannot convert out (type []byte) to type *[256]byte
usr/local/go/src/github.com/klauspost/compress/huff0/decompress.go:1014:20: cannot convert out[dstEvery:] (type []byte) to type *[256]byte
usr/local/go/src/github.com/klauspost/compress/huff0/decompress.go:1015:20: cannot convert out[dstEvery * 2:] (type []byte) to type *[256]byte
usr/local/go/src/github.com/klauspost/compress/huff0/decompress.go:1016:20: cannot convert out[dstEvery * 3:] (type []byte) to type *[256]byte
The command '/bin/sh -c go get github.com/go-debos/debos/cmd/debos &&     go install github.com/go-debos/debos/cmd/debos' returned a non-zero code: 2
```

--- a/config/docker/base/debos.jinja2
+++ b/config/docker/base/debos.jinja2
@@ -1,0 +1,45 @@
+{# SPDX-License-Identifier: LGPL-2.1-or-later
+ #
+ # Copyright (C) 2022 Collabora Limited
+ # Author: Guillaume Tucker <guillaume.tucker@collabora.com>
+-#}
+
+FROM godebos/debos
+MAINTAINER "KernelCI TSC" <kernelci-tsc@groups.io>
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Docker for jenkins really needs procps otherwise the jenkins side fails
+RUN apt-get update && apt-get install --no-install-recommends -y procps
+
+# SSL / HTTPS support
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    apt-transport-https \
+    ca-certificates
+
+# Dependencies to run debos
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    binfmt-support \
+    busybox \
+    debian-ports-archive-keyring \
+    debootstrap \
+    dosfstools \
+    e2fsprogs \
+    linux-image-amd64 \
+    parted \
+    pkg-config \
+    qemu-system-x86 \
+    qemu-user-static \
+    systemd-container \
+    xz-utils
+
+# Jenkins hacks
+RUN useradd -u 996 -ms /bin/sh jenkins
+
+{%- block fragments %}
+{%- for fragment in fragments %}
+{% with is_debian = true, sub_arch = sub_arch %}
+{% include fragment %}
+{%- endwith %}
+{%- endfor %}
+{%- endblock %}

--- a/config/docker/debos.jinja2
+++ b/config/docker/debos.jinja2
@@ -4,41 +4,6 @@
  # Author: Guillaume Tucker <guillaume.tucker@collabora.com>
 -#}
 
-{% extends 'base/debian.jinja2' %}
+{%- include 'base/debos.jinja2' %}
 
-{% block packages %}
-ENV HOME=/tmp
-ENV GOPATH=/usr/local/go
-ENV PATH=$PATH:/usr/local/go/bin
-
-# Dependencies to build debos
-RUN apt-get update && apt-get install --no-install-recommends -y \
-    gcc \
-    git \
-    golang-go \
-    libc6-dev \
-    libostree-dev
-
-# Build debos
-RUN go get github.com/go-debos/debos/cmd/debos
-RUN go install github.com/go-debos/debos/cmd/debos
-
-# Dependencies to run debos
-RUN apt-get update && apt-get install --no-install-recommends -y \
-    binfmt-support \
-    busybox \
-    debian-ports-archive-keyring \
-    debootstrap \
-    dosfstools \
-    e2fsprogs \
-    linux-image-amd64 \
-    parted \
-    pkg-config \
-    qemu-system-x86 \
-    qemu-user-static \
-    systemd-container \
-    xz-utils
-
-# Jenkins hacks
-RUN useradd -u 996 -ms /bin/sh jenkins
-{%- endblock %}
+ENTRYPOINT []

